### ISSUE TITLE
how do I do an image search for amazon linux?

### ISSUE THREAD

**Issue by grimm26** [test-kitchen/kitchen-ec2#254]
I assumed this would work in the platforms section, it does not.

```
  - name: amazon
    image_search:
      owner_id: 137112412989
      name: amazon/amzn-ami-hvm-*.x86_64-gp2
```

It keeps giving me ubuntu, which seems to be the default.

**Comment by whithajess** [comment]
From what I can see the amazon linux images are not supported based on https://github.com/test-kitchen/kitchen-ec2/tree/master/lib/kitchen/driver/aws/standard_platform and from what I tried despite the amazon guide http://docs.aws.amazon.com/opsworks/latest/userguide/cookbooks-101-basics-ec2.html showing differently but it would be good to get some feedback on this from a maintainer

**Comment by davidcpell** [comment]
I've also had trouble using Amazon Linux even when explicitly setting the `image_id`:

```
platforms:
  - name: amazon-linux
    image_id: ami-f303fb93
```

With this setting I still get ubuntu when running `kitchen converge amazon-linux`:

`Detected platform: ubuntu version 16.04 on x86_64. Instance Type: t2.micro. Default username: ubuntu (default).`

**Comment by whithajess** [comment]
Managed to sort out my config to do this see https://github.com/whithajess/wordpress_lamp/blob/master/.kitchen.yml.example

**Comment by davidcpell** [comment]
thanks @whithajess. that did the trick!

**Comment by steven-burns** [comment]
I've created #321 to add support for the Amazon Linux platform.

**Comment by cheeseplus** [comment]
Closed by #321
